java-squid/src/main/java/org/sonar/java/se/ExplodedGraphWalker.java

```diff
@@ -269,6 +268,10 @@ private void visit(Tree tree, Tree terminator) {
       case METHOD_INVOCATION:
         setSymbolicValueOnFields((MethodInvocationTree) tree);
         break;
+      case IDENTIFIER:
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) End this switch case with an unconditional break, return or throw statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.internal.sonarsource.com/coding_rules#rule_key=squid%3AS128)